includes/main.php

```diff
@@ -227,11 +227,17 @@ function wzkb_list_posts_by_term( $term, $level ) {
  */
 function wzkb_article_header( $term, $level ) {
 
-	$output = '
- <h3 class="wzkb_section_name wzkb-section-name-level-' . $level . '">
-  <a href="' . get_term_link( $term ) . '" title="' . $term->name . '" >' . $term->name . '</a>
- </h3>
- ';
+	$wzkb_options = get_option('wzkb_settings',true);
```

Comment:
WordPress CS requires a space before and after the bracket.
Use `wzkb_get_option` to get the option we need.
e.g. `$clickable_section = wzkb_get_option( 'clickable_section', true );`